doc/release-notes/6.9-release-notes.md

```diff
@@ -0,0 +1,325 @@
+
+## API Updates
+
+- A new API endpoint has been added for retrieveing a list of collections to which a given dataset or collection can be linked. See [the guides](https://guides.dataverse.org/en/6.9/api/native-api.html#list-dataverse-collections-to-which-a-given-dataset-or-dataverse-collection-may-be-linked), #11710, and #11741.
+- A new API endpoint has been added to manage dataset licenses. See [the guides](https://guides.dataverse.org/en/6.9/api/native-api.html#update-dataset-license), #11771, #11815, and #11958.
+- A new API endpoint has been added to manage dataset terms of access for restricted files. See [the guides](https://guides.dataverse.org/en/6.9/api/native-api.html#update-dataset-terms-of-access), #11772, and #11893.
+- A new API endpoint has been added for getting and setting the metadata language of a collection. See [the guides](https://guides.dataverse.org/en/6.9/api/native-api.html#list-the-allowed-metadata-languages-of-a-dataverse-collection), 11856, and #11857.
+- The `/{identifier}/templates` endpoint previously required editDataverse permissions to retrieve the list of dataverse templates. This has been corrected. The endpoint now addDataset permissions instead. See #11796 and #11801.
+- The endpoint ``{id}/versions/compareSummary`` was previously returning an incorrect count for
+the ``changedFileMetaData`` field. The logic for calculating this count has been fixed to accurately reflect the total number of file metadata changes across all files in the dataset version. See #11921 and #11944.
+- The "File Version Differences" and "Dataset Version Summaries" API endpoints have been improved with pagination support (with `limit` and `offset` parameters), performance improvements, and a bug fix. See [the guides](https://guides.dataverse.org/en/6.9/api/native-api.html#get-versions-of-a-dataset-with-summary-of-changes), #11561, #11855 and #11859.
+- The Notifications API has been improved (with `onlyUnread`, `limit`, and `offset` parameters) but also has some breaking changes. See [the guides](https://guides.dataverse.org/en/6.8/api/native-api.html#notifications), #11804, #11851, and #11854.
+- The storage API driver endpoint now returns a JSON object with the driver's "name", "type" and "label", and booleans indicating whether the driver has "directUpload", "directDownload", and/or "uploadOutOfBand" enabled. This change also affects the /api/admin/dataverse/{dataverse-alias}/storageDriver api call. In addition, this call now supports an optional ?getEffective=true to find the effective storageDriver (the driver that will be used for new datasets in the collection), See [the guides](https://guides.dataverse.org/en/6.9/admin/dataverses-datasets.html#configure-a-dataverse-collection-to-store-all-new-files-in-a-specific-file-store), #11695, and #11716.
+
+## Security Updates
```

Comment:
Shouldn't the Payara Update be mentioned here?
Also, CORS is kinda security related and now off by default - a hint may be useful.